src/test/java/net/sf/jabref/importer/fileformat/BibtexParserTest.java

```diff
@@ -310,6 +310,40 @@ public void parseRecognizesEntryWhereLastFieldIsFinishedWithComma() throws IOExc
     }
 
     @Test
+    public void parseRecognizesEntryWithAtInField() throws IOException {
+        ParserResult result = BibtexParser.parse(new StringReader("@article{test,author={Ed von T@st}}"));
+
+        List<BibEntry> parsed = result.getDatabase().getEntries();
+        assertEquals(1, parsed.size());
```

Comment:
... which leads to the question whether comments prior the entry are significant to determine "equality" of two entries ;-)

I tend to say "no", thus not modifying the `equals()` method. Other opinions?